src/main/resources/static/js/subscription.js

```diff
@@ -521,7 +521,7 @@ jQuery(document).ready(function () {
         request.responseType = "application/json;charset=utf-8";
         request.onload = function (event) {
             if (this.responseText == "") {
-                window.logMessages("Failed to download template, Error: Could not contact the backend server.");
+                logMessage("Failed to download template, Error: Could not contact the backend server.");
```

Comment:
I would prefer to rewrite this sentence to: "Failed to download template. Could not contact the backend server." Since the jGrowl is already showing this as an error to the user.

Reply:
Done.